src/Entities/Entities.jl

```diff
@@ -10,7 +10,7 @@ module Entities
 
 using Reexport, Dictionaries, StaticArrays
 
-using ..Nodes
+# using ..Nodes
```

Comment:
Minor: Seems that this dependency is needed . So I would suggest to revert the change 
```
Failed to precompile ONSAS [7d8da659-23aa-453a-9a19-a3d1f6c795f5] to "/home/runner/.julia/compiled/v1.11/ONSAS/jl_jfPIrk".
ERROR: LoadError: UndefVarError: `AbstractNode` not defined in `ONSAS.Entities`
```